sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -60,9 +61,14 @@ std::chrono::milliseconds CalculateExponentialDelay(
 {
   if (jitterFactor < 0.8 || jitterFactor > 1.3)
   {
+    double rand;
+    static drand48_data buf;
+    srand48_r(time(NULL), &buf);
+    drand48_r(&buf, &rand);
+
     // jitterFactor is a random double number in the range [0.8 .. 1.3]
     jitterFactor
-        = 0.8 + ((static_cast<double>(static_cast<int32_t>(std::rand())) / RAND_MAX) * 0.5);
+        = 0.8 + (rand / RAND_MAX) * 0.5);
```

Comment:
As far as I see (after reading https://iq.opengenus.org/c-get-random-number/) here I need to remove RAND_MAX at all, because with drand48_r `rand` will always be in range [0.0, 1.0), but previously the `rand` result would be in range [0, RAND_MAX].